src/WidgetName/widget/WidgetName.js

```diff
@@ -126,7 +123,7 @@ define([
                             //TODO what to do when all is ok!
                         },
                         error: dojoLang.hitch(this, function(error) {
-                            console.log(this.id + ": An error occurred while executing microflow: " + error.description);
+                            console.error(this.id + ": An error occurred while executing microflow: " + error.description);
```

Comment:
Console messages are not visible to the user. I think it's better to either remove the error handler completely (the client will then take care of it) or use mx.ui.error to show a message to the user.